did/builder_test.go

```diff
@@ -0,0 +1,138 @@
+package did
+
+import (
+	"testing"
+
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/stretchr/testify/assert"
+)
+
+// Exercise all builder methods
+func TestDIDDocumentBuilder(t *testing.T) {
+	// https://www.w3.org/TR/did-core/#example-did-document-with-1-verification-method-type
+	var exampleAuthenticationEntry = map[string]string{
```

Comment:
please switch these var declarations to use `:=` syntax

Reply:
or add a single `var ( ) ` block

Reply:
sounds good.